stl/inc/yvals_core.h

```diff
@@ -1565,7 +1566,11 @@ _EMIT_STL_ERROR(STL1004, "C++98 unexpected() is incompatible with C++23 unexpect
 
 #if _HAS_CXX20
 #if !defined(__EDG__) || defined(__INTELLISENSE__) // TRANSITION, GH-395
+#if _HAS_CXX23
+#define __cpp_lib_concepts 202207L
+#else // ^^^ C++23 / C++20 vvv
 #define __cpp_lib_concepts 202002L
+#endif // C++20

```

Comment:
Argh - unfortunately, since we still need to wait for EDG to support concepts, we can't move this macro down to the correct place (among the "macros sensitive to language mode"). Could we add a transition comment to the location, maybe?

```cxx
// TRANSITION, GH-395 - move down to language mode sensitive section
```

Reply:
These still need the papers which introduced their value:

```cxx
#if _HAS_CXX23
#define __cpp_lib_concepts 202207L // P2404R3 Move-Only Types For Comparison Concepts
#else // ^^^ C++23 / C++20 vvv
#define __cpp_lib_concepts 202002L // P2408R5 Ranges Iterators As Inputs To Non-Ranges Algorithms
#endif // C++20
```

Reply:
Done!